azurerm/resource_arm_automation_schedule_test.go

```diff
@@ -338,3 +405,113 @@ func checkAccAzureRMAutomationSchedule_recurring_basic(resourceName string, freq
 		resource.TestCheckResourceAttr(resourceName, "timezone", "UTC"),
 	)
 }
+
+func testAccAzureRMAutomationSchedule_recurring_advanced_week(rInt int, location, frequency string, interval int, weekDay string) string {
```

Comment:
>interval int [](start = 100, length = 12)

*suggestion* The only possible value used in your test cases is `1`, so you may hardcode the `1` in the HCL below and remove this parameter. Similar two suggestions below.